src/internal/packager/images/pull_test.go

```diff
@@ -84,6 +85,54 @@ func TestCheckForIndex(t *testing.T) {
 }
 
 func TestPull(t *testing.T) {
+	t.Parallel()
+	testCases := []struct {
+		name        string
+		ref         string
+		expectedErr string
```

Comment:
Often I'll do two different tables for happy and unhappy path. The logic often branches and that can get weird with the test type. This works fine for now though.

Reply:
Yeah agree, when we things to happen with the error or have more to check than it just being there. Keeping this as is for now since it still feels simple